docs/upgrading_to_project_factory_v10.0.md

```diff
@@ -0,0 +1,193 @@
+# Upgrading to Project Factory v10.0
```

Comment:
We also need to note that users will need to remove any references to `skip_gcloud_download` and `use\_tf\_google\_credentials\_env\_var`.

Reply:
good catch.